mp/src/game/client/momentum/ui/SettingsPanel/ComparisonsSettingsPage.cpp

```diff
@@ -23,6 +23,7 @@ ComparisonsSettingsPage::ComparisonsSettingsPage(Panel *pParent)
     m_pCompareShow = new CvarToggleCheckButton(this, "CompareShow", "#MOM_Settings_Compare_Show", "mom_comparisons");
     m_pCompareShow->AddActionSignalTarget(this);
     m_pMaxZones = new CvarTextEntry(this, "Zones", "mom_comparisons_max_zones");
+    m_pMaxZones->SetAllowNumericInputOnly(true);
```

Comment:
Make sure to also do this in the .res files, I believe you'll want `"NumericInputOnly" "1"`

Reply:
I double checked and these text entries have that field. Just felt that it required it in code as well.

Reply:
Nice! All good now.